addon-test-support/frost-link.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Test helpers for frost-link
+ */
+import Ember from 'ember'
+
+/**
+ * Stub the routing service so we can test the route given to a frost-link
+ * @param {*} context - the testing context (`this` from within your beforeEach())
+ */
+export function stubRoutingService (context) {
+  const RouterStub = Ember.Object.extend({
+    generateURL (endpoint, segments) {
+      let url = endpoint
+
+      if (segments.length !== 0) {
```

Comment:
Again, not my code, I just moved it, but I'd only use `isEmpty` if I needed to handle multiple types of data, or multiple possible "empty" values. In this case, `segments` is known to be an array, and so an explicit check on the length is, IMO, clearer and more appropriate.